CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0]
+
+- Increase supported custom integration icon size from 32x32 to 256x256
```

Comment:
The rate limiting changelog entry

Reply:
Ah I guess we missed that. How does this sound: `Added new 'RateLimitError' type to inform the users when they reach their daily usage limit` ?

Reply:
Sounds perfect